server/CMakeLists.txt

```diff
@@ -1,3 +0,0 @@
-add_executable(nupad_server main.cpp)
```

Comment:
Why are the server files removed?

Reply:
Server responsibility is delegated to NSQ, hence there is no need for this module.